src/main/java/seedu/address/ui/UiManager.java

```diff
@@ -20,7 +20,7 @@ public class UiManager implements Ui {
     public static final String ALERT_DIALOG_PANE_FIELD_ID = "alertDialogPane";
 
     private static final Logger logger = LogsCenter.getLogger(UiManager.class);
-    private static final String ICON_APPLICATION = "/images/address_book_32.png";
+    private static final String ICON_APPLICATION = "/images/nutrigoals_32.png";
```

Comment:
I like your choice of the product icon! But since we will not be using `address_book_32.png` anymore, would it be better to have it removed to clear away unused resources?